client/src/shared/Models.ts

```diff
@@ -1,23 +1,33 @@
-export interface BaseArticle {
+export interface BaseCord19Article {
   id: string;
   abstract: string;
   authors: Array<string>;
   journal: string;
   publish_time: string;
+  title: string;
+  source: Array<string>;
+  url: string;
+}
+
+export interface BaseAclArticle {
+  id: string;
+  abstract_html: string;
+  authors: Array<string>;
+  publish_time: string;
   source: Array<string>;
   title: string;
   url: string;
 }
 
-export interface SearchArticle extends BaseArticle {
+export interface SearchArticle extends BaseAclArticle {
   highlights: Array<Array<[number, number]>>;
   highlighted_abstract: boolean;
   paragraphs: Array<string>;
   score: number;
   has_related_articles: boolean;
 }
 
-export interface RelatedArticle extends BaseArticle {
+export interface RelatedArticle extends BaseAclArticle {
```

Comment:
Change this to `Cord19RelatedArticle extends BaseCord19Article` since there is not related articles for ACL